src/app.js

```
@@ -1 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
+
+const args = process.argv.slice(2);
+
+const operation = args[0];
+
+const numbers = args.slice(1).filter(arg => !isNaN(arg)).map(Number);
+
+if (!numbers.length) {
+  switch (operation) {
+    case 'add' || 'multiply':
+      console.log(`There is nothing to ${operation}`);
+      break;
+    default:
+      console.log('Operation is not supported!');
```

It would be great to throw an error instead of console log